### ISSUE TITLE
pytorch

### ISSUE THREAD

**Issue by freakboy3742** [beeware/mobile-forge#7]
### What is PyPI name of the package you would like to see added?

pytorch

### Additional details

Requested by @yousifKashef (freakboy3742/chaquopy#8)

**Comment by marcpabst** [comment]
As I understand from the other issue, this is currently being held back by a missing Fortran compiler for iOS. After a quick Google search, I'm unsure why there is a need for one in the first place. After all, `libtorch` is C++ and apparently compiles for iOS fine. Is it a dependency on LAPACK or BLAS that needs Fortran?

**Comment by mhsmith** [comment]
Yes, PyTorch needs LAPACK or BLAS, but I don't think it needs Fortran directly. 

As mentioned at https://github.com/beeware/mobile-forge/issues/10#issuecomment-1929289223, Apple may already provide official LAPACK and BLAS builds for iOS.

**Comment by marcpabst** [comment]
Right, I didn't catch that. With the new official iOS support for Python, we might be able to spark some interest from the PyTorch developers regarding this.
